lib/AL_USDMaya/AL/usdmaya/DebugCodes.cpp

```diff
@@ -25,6 +25,8 @@ TF_REGISTRY_FUNCTION(TfDebug)
   TF_DEBUG_ENVIRONMENT_SYMBOL(ALUSDMAYA_TRANSLATORS, "UsdMaya translators");
   TF_DEBUG_ENVIRONMENT_SYMBOL(ALUSDMAYA_EVENTS, "UsdMaya events");
   TF_DEBUG_ENVIRONMENT_SYMBOL(ALUSDMAYA_EVALUATION, "UsdMaya evaluation");
+  TF_DEBUG_ENVIRONMENT_SYMBOL(ALUSDMAYA_EVALUATION_BBOX, "UsdMaya bounding box "
+      "evaluation");
```

Comment:
This doesn't need two lines. We've collectively agreed to use dual 4k monitors here, so extending a line of text to 120 chars instead of the 80 chars we used in the 1970's is ok :p